### PR TITLE
network-security/tor update

### PR DESCRIPTION
**Pull request by pnenpbit**
Updated section on Tor. Updated photos, names, sites, and more. Android section also created.

### CODE REVIEW

**Comment by pnenpbit** [comment]
First time using github, excuse any mistakes made during the process of creating this merge.

**Comment by micah** [comment]
Thanks for this, looks like a lot of good work! We'll have a look at it
and get back to you soon if there are any issues.

File: pages/security/network-security/tor/en.text [hunk 1]
```diff
@@ -6,54 +6,63 @@ Welcome to the exciting world of Tor! This page will get you up and running, in
...
 
-!tbb-screenshot1.jpg!
+!tb-install-finished.png!
 
 See the Using Tor section below for how to use Tor.
 
 h3. Mac OS X
 
-After you've downloaded the Tor Browser Bundle, then simply click on it to begin the installation. Mac OS X will automatically unarchive it and you will have a TorBrowser application in your chosen language (for example, TorBrowser_en-US.app). Click the TorBrowser application to launch Vidalia. Once Vidalia connects to Tor, it will launch Firefox.
+After you've downloaded the Tor Browser Bundle, simply click on it to begin the installation. Mac OS X will automatically unarchive it and you will have a TorBrowser application in your chosen language (for example, TorBrowser_en-US.app). Click the TorBrowser application to launch the connectoin wizard. Once connected to Tor, it will launch Tor Browser.
```
Comment (disturbio) [inline pages/security/network-security/tor/en.text]: > connectoin

this is probably a small typo

**Comment by micah** [comment]
>     A pages/security/network-security/tor/tb-desktop-file.png (0)
>     A pages/security/network-security/tor/tb-good-con.png (0)
>     A pages/security/network-security/tor/tb-install-finished.png (0)
>     A pages/security/network-security/tor/tb-start-conwiz.png (0)

The above files are zero bytes, but you added them. Did you forget to
include them?

**Comment by pnenpbit** [comment]
> A pages/security/network-security/tor/tb-desktop-file.png (0) A pages/security/network-security/tor/tb-good-con.png (0) A pages/security/network-security/tor/tb-install-finished.png (0) A pages/security/network-security/tor/tb-start-conwiz.png (0)
> The above files are zero bytes, but you added them. Did you forget to include them?

No the pictures are there. If you'd like, I can re-upload the photos to an image hosting service.
https://share.riseup.net/#9srglbb-F8fEl_TR9f_NWQ

File: pages/security/network-security/tor/en.text (hunk 1 above)
Comment (pnenpbit) [reply pages/security/network-security/tor/en.text]: Yep.

**Comment by micah** [comment]
indeed, you are right, its just in the github email that it shows 0
bytes, that is odd.

**Comment by micah** [comment]
Great, thanks for fixing the typo, I'll merge this now :+1:

**Pull request merged**